annealing.gemspec

```diff
@@ -29,5 +29,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'ruby-prof', '~> 1.4.1'
+  spec.add_development_dependency 'rake', '~> 13.0', '>= 13.0.6'
+  spec.add_development_dependency 'rubocop', '~> 1.23'
```

Comment:
Dependencies are still here

Reply:
Ah, sorry, I overlooked those. Thanks for the reminder. Will fixup asap